src/test/scala/mesosphere/marathon/core/task/update/impl/TaskStatusUpdateProcessorImplTest.scala

```diff
@@ -294,6 +293,26 @@ class TaskStatusUpdateProcessorImplTest extends AkkaUnitTest {
       Then("not do anything else")
       verifyNoMoreInteractions()
     }
+
+    "receiving an running update for known instnace but unknown task" in new Fixture {
+      val appId = PathId("/app")
+      val instance = TestInstanceBuilder.newBuilder(appId).addTaskResidentLaunched().getInstance()
+      val incrementedTaskId = Task.Id.forResidentTask(Task.Id(instance.instanceId.idString))
+      val status = MesosTaskStatusTestHelper.running(incrementedTaskId)
+
+      taskTracker.instance(instance.instanceId) returns Future.successful(Some(instance))
+
+      When("publish the status")
+      updateProcessor.publish(status).futureValue
+      Then("load the task in the task tracker")
+      verify(taskTracker).instance(instance.instanceId)
+      Then("initiate the task kill")
+      verify(killService).killUnknownTask(incrementedTaskId, KillReason.Unknown)
```

Comment:
Wait, I thought we do not to kill the task.

Reply:
The ticket says
> [...]
> When Mesos sends a task running update for a task ID with a launchCount counterpart newer than that which Marathon expects,
> **Marathon kills the task**, and eventually replaces it

Reply:
Should we add a new `KillReason.Outdated`?